doc/k8s/sqlflow-mysql.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
+metadata:
+  name: sqlflow-mysql
+spec:
+  selector:
+    matchLabels:
+      app: sqlflow
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: sqlflow
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+      containers:
+      - image: yancey1989/sqlflow:k8s
```

Comment:
What is the difference between this image and `sqlflow/sqlflow:latest`?

Reply:
Just used to test, change to the official image.